intel/AiCSD
# feat: Implement deletion script for WSL2 port forwarding rules

## PR Checklist

## Author Mandatory (to be filled by PR Author/Submitter)

### CODE MAINTAINABILITY

    - <https://github.com/intel/AiCSD/blob/main/.github/CONTRIBUTING.md>
     - Manual testing required
     -  Adding as follow on PR

### What are the steps to test this PR contribution?

How does a reviewer know it works?

 - Run `WSL_Port_Setup_Script.ps1` using Admin Powershell on OEM Device
 - Check status with the following:
      - `netsh interface portproxy show all`
      - Redirect output of `Get-NetFirewallRule -Direction Inbound -Enabled True` to a file then search for "Open SSH" (alternatively the rules should be at the bottom)
 - Run `WSL_Port_Removal_Script.ps1` using Admin Powershell on OEM Device
 - Check status with the following:
      - `netsh interface portproxy show all` - The rules should be deleted
      - Redirect output of `Get-NetFirewallRule -Direction Inbound -Enabled True` to a file then search for "Open SSH" (alternatively the rules should be at the bottom) - The rules should be deleted

Are there any special instructions beyond standard workflow?

 - This requires Windows 10 and admin privileges to access Powershell as Admin

## Maintainer Mandatory (to be filled by PR Reviewer/Approving Maintainer)

### QUALITY CHECKS


## Other information
